docs/manuals.ja/index.md

```diff
@@ -196,8 +196,6 @@ REST API を介したデータのクエリと取得が計画されています
     "Grafana Home"
 [grafana.pg]: http://docs.grafana.org/features/datasources/postgres/
     "Grafana PostgreSQL Data Source"
-[influx]: https://docs.influxdata.com/influxdb
```

Comment:
shouldn't we leave this to @fisuda? not sure if he also mentioned influx db in the japanese docs?